### PR TITLE
Add docs for Scout Web App

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

File: _data/toc.yaml [hunk 1]
```diff
@@ -1885,6 +1885,8 @@ manuals:
       title: Image details view
     - path: /scout/advanced-image-analysis/
       title: Advanced image analysis
+    - path: /scout/web-app/
```
Comment (fdiogo) [inline _data/toc.yaml:1888]: **issue:** does this change the path of the docs? If so we'd have to deploy a change in scout.docker.com to reflect that

Comment (ChrisChinchilla) [reply _data/toc.yaml:1888]: @fdiogo Yes, but I can also add a redirect, depends which you think is quicker / easier?

Comment (fdiogo) [reply _data/toc.yaml:1888]: Could we add a redirect and tomorrow we deploy the url change?

Comment (ChrisChinchilla) [reply _data/toc.yaml:1888]: @fdiogo Done

File: scout/images/webapp-suggested-fix.png
[the remark this answers is not in the document]
Comment (ChrisChinchilla) [reply scout/images/webapp-suggested-fix.png:1]: OK, I hadn't noticed, but yes, let's do that then

[the remark this answers is not in the document]
Comment (ChrisChinchilla) [reply scout/images/webapp-suggested-fix.png:1]: Actually, it only took a second, so updated

**Review by fdiogo — APPROVED** [review]

File: scout/web-app.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+description: The Docker Scout Web App helps review and share the analysis of images.
+keywords: scanning, analysis, vulnerabilities, Hub, supply chain, security, report, reports, web app
+title: Web App
```
Comment (dvdksn) [inline scout/web-app.md:4]: Have you discussed using a less generic product name for this? For example, Cockpit to keep with the nautical theme? ⛵ 

> A cockpit is a name for the location of controls of a vessel

Comment (fdiogo) [reply scout/web-app.md:4]: 🌶️🌶️🌶️

This will definitely stir the pot 😄

**Review by fdiogo — DISMISSED** [review]

File: scout/image-details-view.md [hunk 3]
```diff
@@ -17,6 +17,8 @@ perspectives. The view displays vulnerabilities and packages that an image
 contains. You can choose whether you want to see data for the image as a whole,
 or for a specific base image or layer.
 
+<!-- TODO: Update image with site link -->
```
Comment (aevesdocker) [inline scout/image-details-view.md:20]: 👀

File: docker-hub/vulnerability-scanning.md [hunk 4]
```diff
@@ -9,6 +9,8 @@ title: Basic vulnerability scanning
 > Hub Vulnerability Scanning requires a
 > [Docker Pro, Team, or Business subscription](../subscription/index.md).
 
+<!-- TODO: Add screenshot and description based on image 2 -->
```
Comment (aevesdocker) [inline docker-hub/vulnerability-scanning.md:12]: 👀

File: scout/web-app.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+> **Note**
+>
+> You can find more details on the elements in the image layer view in [the image details view docs](./image-details-view.md).
+
+Click the **View recommended fixes** button to see instructions to apply the recommended fixes for the image.
+
+![Screenshot of recommended fixes for an image](./images/webapp-suggested-fix.png)
+
+## Packages & dependencies
```
Comment (aevesdocker) [inline scout/web-app.md:66]:
```suggestion
## Packages and dependencies
```

File: scout/web-app.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+- The license(s) used by the package.
+- The versions of the package used by images in the organization.
+- The package type.
+- The number of images that use the package.
+
+## Vulnerabilities
+
+![Screenshot showing Docker Scout Vulnerabilities list](./images/webapp-vulns.png)
+
+The vulnerabilities tab shows a list of all vulnerabilities from images in the organization. You can sort the list by severity and search for Common Vulnerabilities and Exposures (CVE) ID using the search box.
```
Comment (aevesdocker) [inline scout/web-app.md:85]:
```suggestion
The **Vulnerabilities** tab shows a list of all vulnerabilities from images in the organization. You can sort the list by severity and search for Common Vulnerabilities and Exposures (CVE) ID using the search box.
```

File: scout/web-app.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+Each entry in the list shows the following details:
+
+- Severity of the vulnerability.
+
+  > **Note**
+  >
+  > The calculation behind this severity level comes from a variety of sources
+  > that Docker Scout collates based xx.
```
Comment (aevesdocker) [inline scout/web-app.md:94]: based on....?

File: scout/image-details-view.md (hunk 3 above)
Comment (ChrisChinchilla) [reply scout/image-details-view.md:20]: @aevesdocker Yeah, I left them on purpose, but I think I can now update them

Comment (ChrisChinchilla) [reply scout/image-details-view.md:20]: Actually this is one I can't update yet until DD updates…

**Review by aevesdocker — APPROVED** [review]

**Review by aevesdocker — APPROVED** [review]

**Pull request merged**